config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'dishes/show'
```

Comment:
Either way, there's better Rails syntax for this. we can do:
`resources :dishes, only: [:show]` and that should give you the same routes you have now, plus a helper.